speakeasy/winenv/api/kernelmode/ntoskrnl.py

```diff
@@ -2732,6 +2897,7 @@ def ZwWriteFile(self, emu, argv, ctx={}):
         );
         """
         FileHandle, evt, apc, apc_ctx, ios, buf, length, offset, key = argv
+        length = length & 0xFFFFFFFF
```

Comment:
This bug is likely everywhere in the code and can probably be used to crash the emulator in more places.